src/main/java/Main.java

```diff
@@ -0,0 +1,43 @@
+import java.io.File;
+
+import org.apache.catalina.WebResourceRoot;
+import org.apache.catalina.core.StandardContext;
+import org.apache.catalina.startup.Tomcat;
+import org.apache.catalina.webresources.DirResourceSet;
+import org.apache.catalina.webresources.StandardRoot;
+
+public class Main {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All classes, interfaces, enums and annotations must belong to a named package](https://app.codacy.com/app/thejerome/IFMO_Servlets_20190915/pullRequest?prid=4297218)